po/pl.po

```diff
@@ -1,6 +1,6 @@
 # Polish translations for Screen Recorder.
 # This file is distributed under the same license as the com.github.mohelm97.screenrecorder.
-# Translated by Michał Nowakowski <nowakowski@offshore.rocks>, 2019.
+# Michał Nowakowski <nowakowski@offshore.rocks>, 2019.
 #
 #, fuzzy
```

Comment:
```suggestion

```

Reply:
Didn't notice before but as long as there are no more `fuzzy` strings in the file you have to remove the `fuzzy` line from the header too.